### PR TITLE
Adding monitoring priority to all triggers.

### PR DESCRIPTION
**Pull request by jjudd**
Triggers now have an associated monitoring priority. 

This priority is used to determine whether or not to send StatsD metrics for a trigger. It is persisted in the database and can be accessed via a model. Accordingly, it can be used for additional monitoring beyond the monitoring built into Piezo. For example, a monitoring job could be created and the priority used to determine whether an alert should be created, and what priority it should be when a trigger enters into an error state or fails to run.

### CODE REVIEW

**Comment by gregghz** [comment]
LGTM.

Maybe we should have peizo depend on relate to clean up / make the jdbc stuff a bit safer and more readable.

**Comment by jjudd** [comment]
Agreed. That and a real patch system for the db patches. Both are outside the scope of this PR, though.

File: worker/src/main/resources/piezo_mysql_5.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO trigger_monitoring_priority (trigger_name, trigger_group, priority) 
+SELECT TRIGGER_NAME, TRIGGER_GROUP, 3 from QRTZ_TRIGGERS;
```
Comment (sriraamas) [inline worker/src/main/resources/piezo_mysql_5.sql]: so we are going to assign highest priority to all triggers in the beginning ?

File: worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala [hunk 2]
```diff
@@ -7,12 +7,13 @@ import org.slf4j.LoggerFactory
...
-  actual_start:Date,
-  finish:Date,
-  misfire:Int
+  name: String,
+  group: String,
+  scheduled_start: Date,
+  actual_start: Date,
+  finish: Date,
+  misfire: Int,
+  fire_instance_id: String
```
Comment (sriraamas) [inline worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala]: why exactly do we need `fire_instance_id`?

Comment (jjudd) [inline worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala]: I originally missed that triggers were stored in the job_history table, so I exposed this member as a way to be able to look up jobs in job_history from information in trigger_history. It is no longer needed, but it is part of the table, and I don't see any harm in exposing it.

**Comment by disaacson** [comment]
Looks good! Just the couple of noted tweaks.

**Pull request merged**